staging/src/github.com/kubeedge/mapper-framework/hack/make-rules/generate.sh

```diff
@@ -27,6 +33,13 @@ function entry() {
     exit 1
   fi
   cp -r "${ROOT_DIR}/_template/mapper" "${mapperPath}"
+  if [ "${buildMethod}" = "stream" ]; then
+    rm "${mapperPath}/data/stream/handler_nostream.go"
+  fi
+
+  if [ "${buildMethod}" = "nostream" ]; then
+      rm "${mapperPath}/data/stream/handler.go" "${mapperPath}/data/stream/img.go" "${mapperPath}/data/stream/video.go"
+  fi
```

Comment:
If add new file for stream, this part will be hard to maintain.

Reply:
I modified it to only keep the handler_stream.go file and renamed this file to handler.go after build